keyboards/dztech/dz65rgb/keymaps/adi_dz65rgb_ansi/readme.md

```diff
@@ -0,0 +1,13 @@
+# Adi's DZ65RGB Layout
+[Layout Visual](https://i.imgur.com/xuFkvVI.png)
```

Comment:
**OPTIONAL**

This change will make GitHub render the image in the page, if you want.

```suggestion
![Layout Visual](https://i.imgur.com/xuFkvVI.png)
```